metrics/lastseen/export.go

```diff
@@ -0,0 +1,36 @@
+package lastseen
+
+import "context"
+
+// Exporter defines a last seen metrics exporter
+type Exporter interface {
+	Flush(ctx context.Context, field string, val LastSeen) error
+}
+
+// Multi returns an exporter that invokes all the exporters given to it in order.
+func Multi(e ...Exporter) Exporter {
+	a := make(multi, 0, len(e))
+	for _, i := range e {
+		if i == nil {
+			continue
+		}
+		if i, ok := i.(multi); ok {
+			a = append(a, i...)
+			continue
+		}
+		a = append(a, i)
+	}
+	return a
+}
+
+type multi []Exporter
+
+func (m multi) Flush(ctx context.Context, field string, val LastSeen) error {
+	for _, o := range m {
```

Comment:
I'd rename the `o` variable to `exporter` : it took me a while what was this `o` thing :)

Reply:
o !